Sources/src/ExtMonFNALMARSUtils.cc

```diff
@@ -75,46 +75,46 @@ namespace mu2e {
     PDGCode::type MARSMu2eConverter::marsToMu2eParticleCode(int marsPID) {
       static std::map<int,int> table;
       if(table.empty()) {
-        table[1] = +2212; // proton
-        table[2] = +2112; // neutron
-        table[3] = +211; // pi+
-        table[4] = -211; // pi-
-        table[5] = +321; // K+
-        table[6] = -321; // K-
-        table[7] = -13; // mu+
-        table[8] = +13; // mu-
-        table[9] =  22; // gamma
-        table[10] = -11; // e-
-        table[11] = +11; // e+
-        table[12] = -2212; // p- (antiproton)
-        table[13] = 111; // pi0
-        table[14] = 1000010020; // deutron
-        table[15] = 1000010030; // H3
-        table[16] = 1000020030; // He3
-        table[17] = 1000020040; // He4
-        table[18] = +14; // numu
-        table[19] = -14; // numubar
-        table[20] = +12; // nue
-        table[21] = -12; // nuebar
-        table[22] = 130; // K0L
-        table[23] = 310; // K0S
+        table[1] = PDGCode::proton; // proton
+        table[2] = PDGCode::n0; // neutron
+        table[3] = PDGCode::pi_plus; // pi+
+        table[4] = PDGCode::pi_minus; // pi-
+        table[5] = PDGCode::K_plus; // K+
+        table[6] = PDGCode::K_minus; // K-
+        table[7] = PDGCode::mu_plus; // mu+
+        table[8] = PDGCode::mu_minus; // mu-
+        table[9] = PDGCode::gamma; // gamma
+        table[10] = PDGCode::e_plus; // e-
+        table[11] = PDGCode::e_minus; // e+
```

Comment:
In ExtMonFNALMarsUtils.cc  e_plus and e_minus are swapped, please fix

Reply:
From PDGCode.hh
        e_minus = 11 ,
        e_plus = -11 ,
which says the code edit was correct, but the comments are wrong, please double-double-check, I might still be reading something wrong..  If you agree, I'll change the comment.  The other question is whether the MARS translation (table indices) is correct.

Reply:
Hi Ray,   the comments are correct (at least consistent with MARS documentation).   The old code was wrong, and this became explicit with the code improvements you made in this PR.    Probably this PR should be merged as is, and we can fix the bug in a separate one (which might also handle the code duplication).

Reply:
I am looking at fixing the duplicate code in the CRY and CORSIKA modules in this PR.  I could also turn these two methods into a either a free-standing function (with a static member) or a little helper class with an initialized member - let me know which you prefer.

Reply:
@gaponenko and @rlcee can any remaining issues here be addressed in a followup PR?  I would like to merge today or tomorrow.

Reply:
ooops - I just saw the comment a little later on the page that says that a follow PR will work.  No need to reply.